python-avd/pyavd/_anta/input_factories/avt.py

```diff
@@ -24,14 +26,17 @@ def create(self) -> list[VerifyAVTSpecificPath.Input] | None:
         """Create a list of inputs for the `VerifyAVTSpecificPath` test."""
         avt_vrfs = self.structured_config.router_adaptive_virtual_topology.vrfs
         path_groups = self.structured_config.router_path_selection.path_groups
-        static_peers: set[str] = set()
+        static_peers: set[IPv4Address] = set()
 
         for path_group in path_groups:
             if not path_group.static_peers:
                 self.logger_adapter.debug(LogMessage.PATH_GROUP_NO_STATIC_PEERS, path_group=path_group.name)
                 continue
             for static_peer in path_group.static_peers:
-                static_peers.add(static_peer.router_ip)
+                if isinstance(ip_address(static_peer.router_ip), IPv4Address):
+                    static_peers.add(IPv4Address(static_peer.router_ip))
+                else:
+                    self.logger_adapter.debug(LogMessage.IPv6_STATIC_PEER, peer=static_peer.router_ip)
```

Comment:
Avoid creating twice the IP object.
```suggestion
                static_peer_ip = ip_address(static_peer.router_ip)
                if isinstance(static_peer_ip, IPv4Address):
                    static_peers.add(static_peer_ip )
                else:
                    self.logger_adapter.debug(LogMessage.IPv6_STATIC_PEER, peer=static_peer.router_ip)
```

Reply:
updated